pkg/operator/starter.go

```diff
@@ -174,15 +159,29 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		),
 		configClient.ConfigV1(),
 		configInformers.Config().V1().ClusterOperators(),
-		operatorClient,
 		versionRecorder,
+	).WithConfigUpgradableController().
+		WithLogLevelController()
```

Comment:
call prepare here directly. No need for the extra `runnableAPIServerControllers` var.

Reply:
fine in a follow-up